doc/releases/changelog-dev.md

```diff
@@ -9,6 +9,10 @@
 * `PauliWord` sparse matrices are much faster, which directly improves `PauliSentence`.
   [#4272](https://github.com/PennyLaneAI/pennylane/pull/4272)
 
+* The `qchem` functions `primitive_norm` and `contracted_norm` are modified to be compatible with
+  higher versions of scipy.
+  [#4321](https://github.com/PennyLaneAI/pennylane/pull/4321)
+
```

Comment:
Minor, should this be a bug fix?

Reply:
I don't think so, everything worked as expected, but at the cost of pinning scipy.

Reply:
👍🏼